### ISSUE TITLE
Implement Radio Button

### ISSUE THREAD

**Issue by NickAcPT** [NickAcPT/Lithium-Issues#3]

**Comment by MiniDigger** [comment]
this is how forge handles it for their gui stuff:
a checkbox is a button that cycles between true and false when you click it
radio buttons are represented by a button that cycles between all states of an enum when you click it
I guess this only really works when you have a small number of buttons in your radio button group, but I guess that isn't that bad.

**Comment by NickAcPT** [comment]
@MiniDigger yeah, I know that. I'm programming since 2012 (I started with vb.net winforms) and I know that.
The think is that circles in Minecraft are rare...
